scripts/build/libc/mingw-w64.sh

```diff
@@ -208,6 +208,8 @@ mingw_w64_main()
         --prefix=${MINGW_INSTALL_PREFIX} \
         --build=${CT_BUILD} \
         --host=${CT_TARGET} \
+        --enable-wildcard \
+        --disable-dependency-tracking \
```

Comment:
I'm fine with the wildcard change but we should keep dependency-tracking enabled

Reply:
I think crosstool-ng always cleans up build directory so there is no benefit in tracking the changes but dropped this change.